keyboards/fjlabs/bolsa65/info.json

```
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "Bolsa65",
+    "url": "https://www.bolsakeyboardsupply.com",
+    "maintainer": "FJLabs",
+    "width": 15,
```

I missed a small thing.

```suggestion
    "width": 16,
```

---

Ah, thought that was total matrix col's, not U width, thanks.